src/Orders.sol

```diff
@@ -30,33 +53,27 @@ abstract contract OrderDestination {
         }
         emit OutputFilled(originChainId, recipient, token, amount);
     }
+
+    /// @notice Send multiple Output(s) of Order(s).
+    /// @custom:emits OutputFilled for each output
+    /// @param outputs - The array of Outputs to fill.
+    function fill(uint256[] memory originChainIds, Output[] memory outputs) external payable {
```

Comment:
so, if we keep this PR, we could do 
- `(uint256[] memory originChainIds, Output[] memory outputs)`
- we could repurpose `chainId` in the Output struct - use it to be `targetChainId` on the origin chain, and `originChainId` on the target chain (this is probably confusing)